steward/src/commands/query/cosmos.rs

```diff
@@ -17,33 +17,41 @@ impl Runnable for Cosmos {
     }
 }
 
+/// Query Cosmos balance
 #[derive(Command, Debug, Parser)]
+#[clap(
+    long_about = "DESCRIPTION \n\n Query Cosmos balance.\n This command queries the Cosmos balance, taking the name of the key as a String."
+)]
 pub struct Balance {
-    free: Vec<String>,
+    key_name: Vec<String>,
```

Comment:
Still using a vector here rather than a plain `String` (and elsewhere in this file).